doc/man3/EVP_PKEY_new.pod

```diff
@@ -172,6 +199,10 @@ The EVP_PKEY_new_raw_private_key_ex(),
 EVP_PKEY_new_raw_public_key_ex() and
 EVP_PKEY_new_CMAC_key_ex() functions were added in OpenSSL 3.0.
 
+The documentation of B<EVP_PKEY> was amended in OpenSSL 3.0 to allow there to
+be the private part of the keypair without the public part, where this was
+previously implied to be disallowed.
```

Comment:
Maybe add something like: `implied to be disallowed although the implication was not enforced in some cases.`

Reply:
Not sure why that's necessary...  furthermore, I would say it wasn't enforced, period!  Backends have always been able to disregard that rule.

Reply:
ok